tests/queries/0_stateless/01113_local_dictionary_type_conversion.sql

```diff
@@ -1,29 +1,27 @@
--- Tags: no-parallel
+DROP DATABASE IF EXISTS db_01113;
```

Comment:
Seems like this database is not necessary either. Just use `DB currentDatabase()` in the dictionary declaration, right?